### PR TITLE
Q&M: Fix broken links identified by the Broken links report - fix three XML files

### PR DESCRIPTION
**Pull request by Daidihuang**
Hi @cmastr, @Rick-Anderson, @wadepickett, @tdykstra I have fixed the broken links, please verify, thanks。
https://dev.azure.com/msft-skilling/Content/_workitems/edit/335393/?view=edit

### CODE REVIEW

**Comment by Daidihuang** [comment]
I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.

@dotnet-policy-service agree company="Microsoft"




Thanks

Daidi Huang

C+E Skilling - Pubdesk - Beyondsoft Team

Team Email: ***@***.******@***.***> | Service Hour: 8:00 – 17:00 PST

C+E Skilling Publishing Desk Service<https://review.docs.microsoft.com/en-us/content-production-service/topics/pubdesk-documentation/pubdesk-internal/?branch=main>

________________________________

**Comment by Daidihuang** [comment]
Hi @Rick-Anderson would you please review this PR, thanks in advance!

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Authentication.Cookies.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Authentication.Cookies">
   <Docs>
     <summary>Contains types that support cookie based authentication.</summary>
-    <remarks>For more information about using cookie based authentication, see <see href="https://docs.microsoft.comaspnet/core/security/authentication/cookie">Use cookie authentication without ASP.NET Core Identity</see>.</remarks>
+    <remarks>For more information about using cookie based authentication, see <see href="/aspnet/core/security/authentication/identity-api-authorization?view=aspnetcore-8.0">Use cookie authentication without ASP.NET Core Identity</see>.</remarks>
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Authentication.Cookies.xml:4]:
```suggestion
    <remarks>For more information about using cookie based authentication, see <see href="/aspnet/core/security/authentication/identity-api-authorization">Use cookie authentication without ASP.NET Core Identity</see>.</remarks>
```

remove the version query string

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by Daidihuang** [comment]
@Rick-Anderson I have removed the version query string, please review.

**Comment by Rick-Anderson** [comment]
See the Readme file
API PR's ( api/autoapi/... ) should NOT be made here. Changes to API documentation should be made in the [source repository](https://github.com/dotnet/aspnetcore/tree/main/src) against the triple slash /// comments.

**Pull request closed without merging**